ui/component/common/tabs.jsx

```diff
@@ -8,7 +8,7 @@ import {
   TabPanel as ReachTabPanel,
 } from '@reach/tabs';
 import classnames from 'classnames';
-import { useRect } from '@reach/rect';
+import { useOnResize } from '../../effects/use-on-resize';
```

Comment:
Change to `from 'effects/use-on-resize'`

Reply:
Done